pkg/tfbridge/provider.go

```diff
@@ -1079,15 +1079,15 @@ func (p *Provider) Diff(ctx context.Context, req *pulumirpc.DiffRequest) (*pulum
 	// If there were changes in this diff, check to see if we have a replacement.
 	var replaces []string
 	var replaced map[string]bool
-	var properties []string
+	uniqueProps := map[string]struct{}{}
```

Comment:
Should we do the same thing with `replaces` and `stables`?

Reply:
I am not convinced of the effect of that so will rather hold off from doing it.